packages/datetime/test/common/dayPickerTestUtils.ts

```diff
@@ -14,18 +14,17 @@
  * limitations under the License.
  */
 
-/** @fileoverview test utils for react-day-picker v7 */
+/** @fileoverview test utils for react-day-picker v8 */
```

Comment:
Diff here is a result of deleting existing `dayPickerTestUtils.ts` and renaming `dayPicker8TestUtils.ts`. See https://github.com/palantir/blueprint/pull/7357#discussion_r1994196270